src/object.d

```diff
@@ -4795,3 +4795,153 @@ TTo[] __ArrayCast(TFrom, TTo)(TFrom[] from) @nogc pure @trusted
     foreach (v; s)
         assert(v == cast(short) 0xabab);
 }
+
+// Yields the appropriate printf format token for a type T
+// Indended to be used by miniFormat
+private template getPrintfFormat(T)
```

Comment:
Can we move the new code to `core.internal.assert`?

Reply:
Can't name it `assert` as it's a keyword, thus I went for `dassert`.

Reply:
Sure, thanks!